utils/default-helpers/src/main/scala/pl/touk/nussknacker/engine/util/functions/date.scala

```diff
@@ -1,18 +1,24 @@
 package pl.touk.nussknacker.engine.util.functions
 
-import pl.touk.nussknacker.engine.api.Documentation
+import pl.touk.nussknacker.engine.api.{Documentation, HideToString}
 
 import java.time._
 import java.time.temporal.Temporal
 
 object date extends DateUtils(Clock.systemDefaultZone())
 
-class DateUtils(override protected val clock: Clock) extends DateConversions with DateConstants with DateRangeChecker {
+class DateUtils(override protected val clock: Clock) extends DateConversions with DateConstants with DateRangeChecker with HideToString {
 
   @Documentation(description = "Returns current time as an Instant")
   def now(): Instant = Instant.now(clock)
 
-  // We can't has one now() method with multiple overloaded options because ZoneOffset extends ZoneId and we can get
+  @Documentation(description = "Returns current date-time as a LocalDateTime (without a time-zone)")
+  def localDateTime(): LocalDateTime = LocalDateTime.now(clock)
+
+  @Documentation(description = "Returns current date as a LocalDate (without a time-zone)")
+  def today(): LocalDate = LocalDate.now(clock)
```

Comment:
> So far we were trying to avoid having too many methods using default time zone
Makes sense to me, let's stick with that :) I've removed these new methods.